index.js

```diff
@@ -87,9 +87,13 @@ function processLastItem(/* CODE HERE */) {
  * [2] Invoking `processSum` passing `[]` and `(num) => num + 1000`,
  * should return 1000.
 */
-function processSum(/* CODE HERE */) {
-  /* CODE HERE */
-}
+function processSum(numberList, callback) {
+  let sum = 0;
+  for(i = 0; i < numberList.length; i++) {
+    sum += numberList[i];
+  };
+  return callback(sum);
+};
 
```

Comment:
This is a great solution because of how the `+=` assignment operator works.  You could have also used the `.reduce` method.